pkg/command/plugin.go

```diff
@@ -458,6 +463,7 @@ func newSyncPluginCmd() *cobra.Command {
 	var syncCmd = &cobra.Command{
 		Use:   "sync",
 		Short: "Sync the plugins",
```

Comment:
How about something like this?
```suggestion
		Short: "Installs all recommended plugins from the active contexts",
```